src/backend/src/routes/users.routes.ts

```diff
@@ -20,5 +20,6 @@ userRouter.post(
 userRouter.post('/:userId/change-role', isRole(body('role')), validateInputs, UsersController.updateUserRole);
 userRouter.post('/auth/login', UsersController.logUserIn);
 userRouter.post('/auth/login/dev', UsersController.logUserInDev);
+userRouter.post('/:projectId/favorite', body('favorite').isBoolean(), validateInputs, UsersController.updateUserFavorites);
```

Comment:
ok this is kinda reid's fault for writing the ticket lol but really this should probably be on the projects router and should be /:wbsNum/favorite since that's what all the others are

and then yeah peyton is probs right we don't need the body parameter at all we should just flip the user's current status for this project (for example, if 1.1.0 is not favorited and this endpoint gets called, it becomes favorited for the user. if it gets called again, it gets unfavorited)